README.md

```diff
@@ -1,9 +1,11 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Title
-
-> Description of the project
```

Comment:
- [ ] [OPTIONAL] I would highly suggest you personalize your MIT license. As you can see in this image I highlighted places that need to be changed or removed.

<img width="1058" alt="Screen Shot 2022-04-11 at 7 46 28 PM" src="https://user-images.githubusercontent.com/69011963/162765818-1ce37ae6-1218-466e-b73c-833dbc3000c6.png">
.